client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -42,31 +42,46 @@
 
 
     /** controllerURI The controller rpc URI. This can be of 2 types
-     1. tcp://ip1:port1,ip2:port2,...
-        This is used if the controller endpoints are static and can be directly accessed.
-     2. pravega://ip1:port1,ip2:port2,...
-        This is used to autodiscovery the controller endpoints from an initial controller list.
+     * 1. tcp://ip1:port1,ip2:port2,...
+     *    This is used if the controller endpoints are static and can be directly accessed.
+     * 2. pravega://ip1:port1,ip2:port2,...
+     *   This is used to autodiscovery the controller endpoints from an initial controller list.
+     *
+     * @param controllerURI The controller RPC URI.
+     * @return The controller RPC URI.
     */
     private final URI controllerURI;
 
     /**
      * Credentials to be passed on to the Pravega controller for authentication and authorization.
+     *
+     * @param credentials Pravega controller credentials for authentication and authorization.
```

Comment:
Why do you need `@param` and `@return` for fields?

Reply:
Is this because of Lombok? Isn't there a setting in Lombok to sort this out?

Reply:
Yes and this seems to be the only workaround available at this point. Please look at this comment: https://github.com/pravega/pravega/issues/3373#issuecomment-655714531